app/static/src/app/components/options/NumericInput.vue

```diff
@@ -63,8 +83,22 @@ export default defineComponent({
             const cleanedValue = newVal.replace(/,/g, "");
             const numeric = parseFloat(cleanedValue);
             if (!Number.isNaN(numeric)) {
-                lastNumericValueSet.value = numeric;
-                emit("update", numeric);
+                // min max validation
+                if (numeric <= props.max && numeric >= props.min) {
```

Comment:
Could put the conditional clauses in an if...else if..else structure to just make clear that it's going to hit one of them

Reply:
done! definitely a bit more readable!